express-notes-api/index.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+const dataJSON = require('./data.json');
+
+app.get('/api/notes', (req, res) => {
+  const notesJSON = [];
+  for (const key in dataJSON.notes) {
+    notesJSON.push(dataJSON.notes[key]);
+  }
+  if (notesJSON.length > 0) {
+    res.status(200).send(notesJSON);
+  } else {
+    res.send(notesJSON);
+  }
```

Comment:
Since we're doing the same thing in both cases here, we can remove the `if/else`